### PR TITLE
add context.proxy_version into mixer filter attributes

### PR DESCRIPTION
**Pull request by bianpengyuan**
Add context.proxy_version mixer attribute. The installer PR is out: https://github.com/istio/installer/pull/307

### CODE REVIEW

**Comment by kyessenov** [comment]
Hmm, don't we want envoy version there instead of whatever pilot sends?

**Comment by kyessenov** [comment]
/lgtm
`node.IstioVersion` is extracted from `node.Metadata`.

File: pilot/pkg/model/context.go [hunk 1]
```diff
@@ -171,6 +171,14 @@ func (pversion *IstioVersion) Compare(inv *IstioVersion) int {
 	return 0
 }
 
+// ToString converts IstioVersion to a semver format string.
+func (pversion *IstioVersion) ToString() string {
```
Comment (costinm) [inline pilot/pkg/model/context.go]: If this is mixer-specific - I would prefer that it is in mixer package.

Comment (costinm) [reply pilot/pkg/model/context.go]: But not a huge deal - just worried that when mixerv2 and wasm show up we'll be left with dead stuff in pilot.

Comment (bianpengyuan) [reply pilot/pkg/model/context.go]: Moved to mixer plugin.

**Pull request merged**